homeassistant/components/hyperion/strings.json

```diff
@@ -0,0 +1,55 @@
+{
+  "config": {
+    "flow_title": "Hyperion Ambilight",
+    "step": {
+      "user": {
+        "description": "Set up your Hyperion Ambilight with Home Assistant",
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "port": "[%key:common::config_flow::data::port%]"
+        }
+      },
+      "auth": {
+        "description": "Configure authorization to your Hyperion Ambilight server",
+        "data": {
+          "create_token": "Automatically create new token",
+          "token": "Or provide pre-existing token"
+        }
+      },
+      "create_token": {
+        "description": "Choose **Submit** below to request a new authentication token. You will be redirected to the [Hyperion UI]({hyperion_url}) to approve the request. Please verify the shown id is \"{auth_id}\"",
```

Comment:
Seems confusing to make this a link since there is already an external step button that they should press.